components/Cards/SchoolCard/SchoolCard.js

```diff
@@ -33,145 +33,152 @@ const LabelWithScreenReader = ({ isActive, label }) => (
+
+      <div className={styles.cardBlock}>
+        <span className={styles.cardBlockTitle}>Availability</span>
+        <div className={styles.badgeGroup}>
+          <Badge
+            label={<LabelWithScreenReader label="Online" isActive={hasOnline} />}
+            icon={
+              <OnlineIcon data-testid={hasOnline ? 'School has online' : 'School has no online'} />
+            }
+            className={badgeClassNames(hasOnline)}
+          />
+          <Badge
+            label={<LabelWithScreenReader label="Campus" isActive={!hasOnlyOnline} />}
+            icon={<CampusIcon />}
+            className={badgeClassNames(!hasOnlyOnline)}
+          />
+          <Badge
+            label={<LabelWithScreenReader label="Housing" isActive={hasHousing} />}
+            icon={<HousingIcon />}
+            className={badgeClassNames(hasHousing)}
+          />
+          <Badge
+            label={<LabelWithScreenReader label="Equipment" isActive={hasHardwareIncluded} />}
+            icon={<EquipmentIcon />}
+            className={badgeClassNames(hasHardwareIncluded)}
+          />
         </div>
+      </div>
+
+      <div className={styles.cardBlock}>
+        <span className={styles.cardBlockTitle}>Accepts GI Bill</span>
+        {hasGiBill ? 'Yes' : 'No'}
+      </div>
+
+      <div className={styles.cardBlock}>
+        <span className={styles.cardBlockTitle}>VET TEC Approved</span>
+        {hasVetTec ? 'Yes' : 'No'}
+      </div>
+
+      <div className={styles.cardBlock}>
+        <span className={styles.cardBlockTitle}>Campus Locations</span>
+        {getSchoolLocationText(hasOnlyOnline, locations)}
+        {locations.length > 1 && (
+          <Button
+            analyticsObject={{
+              action: 'Button Selected',
+              category: 'Interactions',
+              label: `${name} | Locations`,
+            }}
+            onClick={toggleModalClick}
+            className={styles.modalToggler}
+          >
+            view all
+          </Button>
+        )}
+      </div>
+
+      <div className={styles.cardBlock}>
+        <LinkButton
+          analyticsEventLabel={`${name} | Website`}
+          href={website}
+          fullWidth
+          theme="secondary"
+        >
+          Visit Website
+        </LinkButton>
+      </div>
+    </Card>
+  );
+};
 
-        <div className={styles.cardBlock}>
-          <span className={styles.cardBlockTitle}>Campus Locations</span>
-          {getSchoolLocationText(props.hasOnlyOnline, props.locations)}
-          {props.locations.length > 1 && (
-            <Button
-              analyticsObject={{
-                action: 'Button Selected',
-                category: 'Interactions',
-                label: `${props.name} | Locations`,
-              }}
-              onClick={this.toggleModalClick}
-              className={styles.modalToggler}
-            >
-              view all
-            </Button>
-          )}
-        </div>
+SchoolCard.propTypes = {
```

Comment:
Can you move these above the function definition?

Reply:
I receive the following error when I move them above the function definition:
"ReferenceError: Cannot access 'SchoolCard' before initialization". This is solved when I create functions using "function" keyword instead of arrow functions. Should I switch to "function"?

Reply:
You'll have to change from an arrow function to a vanilla function. Then you should be good to go.

Reply:
Sure. On it!